Source/Particles/MultiParticleContainer.cpp

```diff
@@ -609,6 +626,40 @@ MultiParticleContainer::doFieldIonization ()
     } // pc_source
 }
 
+void
+MultiParticleContainer::doCoulombCollisions ()
+{
+    BL_PROFILE("MPC::doCoulombCollisions");
+
+    for (int i = 0; i < ncollisions; ++i)
+    {
+        auto& species1 = allcontainers[ allcollisions[i]->m_species1_index ];
+        auto& species2 = allcontainers[ allcollisions[i]->m_species2_index ];
+
+        // Enable tiling
+        MFItInfo info;
+        if (Gpu::notInLaunchRegion()) info.EnableTiling(species1->tile_size);
+
+        // Loop over refinement levels
+        for (int lev = 0; lev <= species1->finestLevel(); ++lev){
+
+            // Loop over all grids/tiles at this level
+#ifdef _OPENMP
+            info.SetDynamic(true);
+            #pragma omp parallel
```

Comment:
I'm not sure what it does.

Reply:
I agree with @ax3l here. 
As mentioned by @WeiqunZhang on Slack, right now `Gpu::notInLaunchRegion()` is generally `false` when running on GPU, so it is good practice to add it here. 
(However, note that, when compiling for GPU we turn `OpenMP` off, so that it  does not actually matter here.)